### ISSUE TITLE
Fonts are outlined and have the wrong color

### ISSUE THREAD

**Issue by Spikeone** [Return-To-The-Roots/s25client#563]
RttR always uses outlining, S2 only did that when building names / percents are shown (as they use the small, only capslock font). 

Also RttR uses the wrong colors, we use yellow player colors (index 132 - 135).
S2 uses index 1 - 4 instead. (ffff00, ffdb17, ffbf33, fb9f4b).

Left RttR, right S2
![font](https://cloud.githubusercontent.com/assets/1276342/17463302/2e96a33a-5cc3-11e6-94c3-0327c4af1ee2.png)

**Comment by Flow86** [comment]
we use the outline font always since the non outlined font is very diffuse on large screens.

but the color is a bug yes

**Comment by Spikeone** [comment]
But may we add an option to disable that outlining? It just looks better, even on a 24" screen - would be nice :)

**Comment by Flamefire** [comment]
Are there multiple colors for the font?

**Comment by Flamefire** [comment]
I think the problem is our color filling (related to #542) and I'm afraid this is not really solvable. S2 uses a palette so it replaces each color by another one. We replace the colors using a special palette by grey and the colorize that grey with a given color.

**Comment by Spikeone** [comment]
Violett maps to: bb97e7, 9b77bf, 7b5b9b, 5f4377
Green maps to: 63f7af, 43c373, 2b8f3f, 175f1b
Orange maps to: ffbb47, df9327, c36b0f, a74b00
Grey maps to: 777777, 4f4f4f, 272727, 000000
Red maps to: ff0000, d70000, af0000, 870000
Yellow maps to: ffff00, cbcb00, 979700, 676700
Blue maps to: 00d3f7, 007bdb, 0037bf, 0000a7

But sadly thats the problem: Yellow for a font maps to index 1-4 but yellow for a player maps to 132-135.

**Comment by Flamefire** [comment]
Hm. Strange. It seems like for everything but the font we can indeed get the correct colors by multiplication with a grey value.
As I understand it, there are 4 special colors that are replaced by the player colors, correct? Are those special colors maybe different than the ones used for the fonts? But they grey font is the same (pure grey) so I doubt that.
We could use a special color map for fonts too, but that would make non-yellow fonts different too. May simply don't fix this?

**Comment by Spikeone** [comment]
Maybe this is dependant on the palette used - maybe player colors differ depending on the terrain (as some colors do) or maybe I slightly changed colors when loading the screenshots in gimp. Gonna check that at home. 
Also sadly I it looks like I'm the only one that prefers the original style over our current style so - yes maybe it's not worth fixing :(

**Comment by Flamefire** [comment]
The colors seem to be right. Just checked the palette itself.
It is just possible they use a different base color or so...
Well the problem is more of the work vs the profit ;)

**Comment by lvanderwall** [comment]
@Spikeone 
> [..]Also sadly I it looks like I'm the only one that prefers the original style over our current style[..]

Not quite, I also like the "warmer" text color of the original... but if it's really hard to fix, then perhaps it shouldn't be top priority :)

**Comment by Flamefire** [comment]
Well Point is: We have the following texts (actual values in screenshot):
Yellow    Brown      Grey
FFFF00   592C10   4D554D
FFDB14  714520   696D69
FFBE30   8A6134   828A82

Nearest palette colors
Yellow    Brown      Grey
FFFF00   5B2F13   4F574F
FFDB17  734723   6B6F6B
FFBF33   8B6337   838B83

We need a factor for each color part and row such that yr*r/255=FF, br*r/255=5B, gr*r/255=4F (and so on) So we need to multiply a fixed color (the text color) by the "player"-color to yield the above values.
This is however impossible because e.g. the red part for yellow is constant, but for the others it is variable. Even worse: The green part decreases for yellow while it increases for the others.

See this image: 
![s2colorfont](https://cloud.githubusercontent.com/assets/309017/17561814/0cbd1278-5f28-11e6-9082-849eb7accc86.png)

So solutions:

- Use fixed font colors with 1 pre-colored texture per font (we can use the palette for that)
- Optimize for 1 color and have other colors different
- Find a different approach for coloring via OGL

So I don' think this is possible to solve in a good-for-all way.
